language/move-vm/transactional-tests/tests/module_publishing/republish_module_skip_compatible_linking_hack_struct.exp

```diff
@@ -1 +1,28 @@
 processed 5 tasks
+
```

Comment:
Is this test change related?

Reply:
Wow thats bad. Just seeing now. Lets followup on this.